### ISSUE TITLE
Read Only field values not exporting

### ISSUE THREAD

**Issue by dbooth2** [Esri/Survey123Community#122]
Any fields that are marked as "yes" in the readonly field on the .xls form appear correct in the survey (i.e. values show up and they are read only) however when the form data is exported none of the values in fields marked read only are exported - instead they're all null. When exporting to a file geodatabase, not only are the field values null, but I cannot find a way to change the read-only field status in the file geodatabase, so I have to create all new fields and enter in the values that did not get exported manually. Can you take a look at this? Thanks.

**Comment by survey123** [comment]
@dbooth2 
This issue is caused at publish time.  Any question in your Survey flagged as read-only was mapped into a geodatabase field which we flagged also as read-only. Turned out not to be a good idea. From now we will not make geodatabase fields read-only. Adding to our International Beer Day milestone in early August for Connect. For this change to take effect you will need to publish again your Survey once we publish the update.
